### PR TITLE
Add the edge core extension as a dependency.

### PR DESCRIPTION
**Pull request by dompuiu**
## Description

Add the [reactor-extension-core-edge](https://github.com/adobe/reactor-extension-core-edge) extension as a dependency so that it can be used by other developers in conjunction with their own extension.

## Motivation and Context

For client side extension we have created a build in extension (we call it the Sandbox extension) in order to provide a few events (page top, click), actions (log a message to the console) and data elements that can be used by developers to test their extension. 

We did the same for edge extensions. But the added data elements and actions were copying the same functionality that already exists in the [reactor-extension-core-edge](https://github.com/adobe/reactor-extension-core-edge). I decided to add the core edge extension as a dependency instead. We plan some enhancements there, so we don't need to update the same functionality in 2 places.

## How Has This Been Tested?

Tested locally while the sandbox is linked and also while it is installed from an archive.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/tasks/helpers/getExtensionDescriptors.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ module.exports = () => {
 
     const extensionDescriptor = require(resolvedExtensionDescriptorPath);
     extensionDescriptor.extensionDescriptorPath = resolvedExtensionDescriptorPath;
-    extensionDescriptors[extensionDescriptor.name] = extensionDescriptor;
+
+    if (platform === extensionDescriptor.platform) {
```
Comment (brenthosie) [inline src/tasks/helpers/getExtensionDescriptors.js:31]: @dompuiu What is the consequence of having no name here if the platform doesn't match?

File: src/tasks/run.js [hunk 2]
```diff
@@ -290,7 +292,7 @@ const configureApp = (app) => {
   }
 };
 
-if (isSandboxLinked()) {
+if (isSandboxLinked() && !process.env.SKIP_DEV_SERVER) {
```
Comment (brenthosie) [inline src/tasks/run.js:295]: Did you mean to leave this check in, and is it more for internal testing or something to be documented publicly?

**Review by brenthosie — APPROVED** [review]
If you don't have major concerns to these questions, then look good to go.

File: src/tasks/helpers/getExtensionDescriptors.js (hunk 1 above)
Comment (dompuiu) [reply src/tasks/helpers/getExtensionDescriptors.js:31]: @brenthosie You will always have a sandbox extension and the extension you are developing.

File: src/tasks/run.js (hunk 2 above)
Comment (dompuiu) [reply src/tasks/run.js:295]: @brenthosie It is more for internal testing. Sometimes I don't need the dev server to start and in order to make everything faster I was always commenting the line 296. I don't think this needs to be documented. The sandbox is linked only when we develop new features.

**Pull request merged**